monkey/infection_monkey/ransomware/ransomware_payload.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, config: dict, telemetry_messenger: ITelemetryMessenger):
             target_directories["windows_target_dir"]
             if is_windows_os()
             else target_directories["linux_target_dir"]
-        )
+        ).strip()
```

Comment:
If the user accidentally leaves a space before or after the path, the payload won't run at all. But since, like you said, we shouldn't be preventing valid directories from being used, no matter how unusual they might be, how about we add a check in the UI and show a warning message if there are stray spaces?

Reply:
A space before the path will be flagged, since this isn't an absolute path. A space after is trickier, since we don't really have a mechanism to distinguish a warning from something invalid. I say we leave it as is. After all, we don't spell check the directory names to check for typos.

Reply:
Windows doesn't care about space after.

![image](https://user-images.githubusercontent.com/15820737/124607763-6f4cb900-de6e-11eb-9bc2-c593b932e6e4.png)